### PR TITLE
Greg Development

### PR DESCRIPTION
**Pull request by GregVargas1999**
Added the MessageFeedServlet.java file

### CODE REVIEW

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ if (!parameterUsername) {
 function setPageTitle() {
   document.getElementById('page-title').innerText = parameterUsername;
   document.title = parameterUsername + ' - User Page';
+  console.log("IS THIS SHOWING UP OR NAH");
```
Comment (GregVargas1999) [inline src/main/webapp/js/user-page-loader.js]: uh yeah meant to delete this

Comment (gabrielacorona) [reply src/main/webapp/js/user-page-loader.js]: HAHASHAHA

**Review by gabrielacorona — APPROVED** [review]
approved

**Review by ctiscareno — COMMENTED** [review]
Attempting to review code

**Pull request merged**